### PR TITLE
Adding Turkish language support to the theme

### PR DESCRIPTION
**Pull request by ngurmen**
- added "/tr/" sections to all relevant locales.ts files
- added packages/shared/src/node/utils/dayjs/tr.ts for day and month names

closing #2346

### CODE REVIEW

File: packages/shared/src/node/utils/dayjs/tr.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+     LTS: "HH:mm:ss",
+     LT: "HH:mm",
+     L: "DD.MM.YYYY",
+     LL: "D MMMM YYYY",
+     LLL: "D MMMM YYYY HH:mm",
+     LLLL: "dddd, D MMMM YYYY HH:mm",
+  },
+};
+
+export const loadEnLocale = (extendeddayjs: typeof dayjs): void => {
```
Comment (Mister-Hope) [inline packages/shared/src/node/utils/dayjs/tr.ts:28]:
```suggestion
export const loadTrLocale = (extendeddayjs: typeof dayjs): void => {
```

**Review by Mister-Hope — CHANGES_REQUESTED** [review]

**Comment by Mister-Hope** [comment]
You doesnot tick "Allow edits from maintainers", so I am not be able to update your pr.

You need to:

1. Add tr in `vuepress-shared` so that we can recoginze it: https://github.com/vuepress-theme-hope/vuepress-theme-hope/blob/main/packages/shared/src/node/locales/config.ts
2. Add theme locales in https://github.com/vuepress-theme-hope/vuepress-theme-hope/tree/main/packages/theme/src/node/locales and do not foget to add it to index.ts
3. To let dayjs workwell, this file needs an update: https://github.com/vuepress-theme-hope/vuepress-theme-hope/blob/main/packages/shared/src/node/utils/dayjs/index.ts

**Comment by ngurmen** [comment]
@Mister-Hope , thanks for your feedback.  I updated all files.  There is just one file we are still working on.  When finished I will re-request your review.  Just wanted to check "Allow edits from maintainers" but I don't see that check box as shown in the image below.  Not sure why.

image

I did not use this workflow of creating a PR from a fork before.  I thought maybe you have some ideas why that one is not available.   Thanks.

**Comment by Mister-Hope** [comment]
There should be a checkbox when you opening a PR.

File: packages/theme/src/node/locales/tr.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+  encryptLocales: {
+    iconLabel: "Sayfa şifreli",
+    placeholder: "Şifrenizi girin",
+    remember: "Şifreyi hatırla",
+    errorHint: "Lütfen doğru şifreyi girin!",
+  },
+
+  routeLocales: {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
```
Comment (Mister-Hope) [inline packages/theme/src/node/locales/tr.ts:64]:
```suggestion
```

**Comment by Mister-Hope** [comment]
Could you please pass the linter?

**Comment by ngurmen** [comment]
fixed the issues, it should now pass the linter.  thx.

**Pull request merged**